lnwallet/interface.go

```diff
@@ -393,7 +393,7 @@ type WalletController interface {
 	// retrieve the transactions relevant to a specific account. When
 	// empty, transactions of all wallet accounts are returned.
 	ListTransactionDetails(startHeight, endHeight int32,
-		accountFilter string) ([]*TransactionDetail, error)
+		accountFilter string, indexOffset int, maxTnxs int) ([]*TransactionDetail, error)
```

Comment:
nit: break before `maxTnxs` to avoid too long line. Also, should be spelled `maxTxns`?

Reply:
Avoiding a long line was why I named it maxTnx, but after breaking the line, I now renamed it to  maxTransactions

Reply:
Looks like this is still unaddressed?

Reply:
Or I see, I unintentionally addressed this in another [commit](https://github.com/lightningnetwork/lnd/pull/8998/commits/32cdb82c810591f33be63d6fb0b734146231a644#diff-ee18c843be45e96763c5d376abd52914973db78bceb4070414b369b20d8eb06eR397)